pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -53,21 +53,27 @@
   @JsonPropertyDescription("Partial update strategies.")
   private final Map<String, Strategy> _partialUpsertStrategies;
 
+  @JsonPropertyDescription("global upsert strategy")
+  private final Strategy _globalUpsertStrategy;
+
   @JsonPropertyDescription("Column for upsert comparison, default to time column")
   private final String _comparisonColumn;
 
   @JsonCreator
   public UpsertConfig(@JsonProperty(value = "mode", required = true) Mode mode,
       @JsonProperty("partialUpsertStrategies") @Nullable Map<String, Strategy> partialUpsertStrategies,
+      @JsonProperty("globalUpsertStrategy") @Nullable Strategy globalUpsertStrategy,
       @JsonProperty("comparisonColumn") @Nullable String comparisonColumn,
       @JsonProperty("hashFunction") @Nullable HashFunction hashFunction) {
     Preconditions.checkArgument(mode != null, "Upsert mode must be configured");
     _mode = mode;
 
     if (mode == Mode.PARTIAL) {
       _partialUpsertStrategies = partialUpsertStrategies != null ? partialUpsertStrategies : new HashMap<>();
+      _globalUpsertStrategy = globalUpsertStrategy != null ? globalUpsertStrategy : Strategy.OVERWRITE;
```

Comment:
This is changing the existing behavior. I do agree `OVERWRITE` makes more sense to partial upsert, but not sure if we want to introduce backward incompatibility here

Reply:
Thanks @Jackie-Jiang for review. If we use Ignore (https://github.com/apache/pinot/pull/7907) as default behavior for global mergers, do you think it will solve the backward compatible issue.

Reply:
I'm not sure how much adoption of partial uspert today, so I feel it's ok to get to best possible default option, whenever we can